src/Route.php

```diff
@@ -39,6 +42,18 @@ private function normalizePath(string $path): string
 	}
 
 
+	public function supportedHttpMethods(): ?array
+	{
+		return $this->supportedHttpMethods;
+	}
+
+
+	public function allowOnceAllMethods()
```

Comment:
Imo better name for this method would be `allowAllHttpMethods()` as the `Once` word here is useless in my eyes.